CurrentLocationKotlin/.google./packaging.yaml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# GOOGLE SAMPLE PACKAGING DATA
+#
+# This file is used by Google as part of our samples packaging process.
+# End users may safely ignore this file. It has no relevance to other systems.
+---
+status:       PUBLISHED
+technologies: [Android]
+categories:   [Location]
+languages:    [Kotlin]
+solutions:    [Mobile]
+github:       android/location-samples
+level:        INTERMEDIATE
+apiRefs:
+    - gms:com.google.android.gms.location.FusedLocationProviderClient
+    - gms:com.google.android.gms.location.LocationServices
+    - gms:com.google.android.gms.location.LocationRequest.PRIORITY_HIGH_ACCURACY
+    
+license: apache2
```

Comment:
Does this need capitalization, or is lowercase expected?

Reply:
Good question... looking through it... Lowercase is how the template has them and all the other packaging.yaml files I have seen.